### ISSUE TITLE
`get_zone_conditioning_category_dict()` `GET_ZONE_CONDITIONING_CATEGORY_DICT__REQUIRED_FIELDS`

### ISSUE THREAD

**Issue by JacksonJ-KC** [pnnl/ruleset-checking-tool#1446]
The fields listed as required fields for this function are controversial. In several rules we allow `design_sensible_cool_capacity` and `rated_sensible_cool_capacity` to be used interchangeably because we are not confident which (or if both) will be able to be populated by RPD generation software. 

Are we sure that we want these to be required fields that result in UNDETERMINED via MissingKeyException if any are not defined when `get_zone_conditioning_category_dict()` is called?

```
"building": {
        "building_segments[*].heating_ventilating_air_conditioning_systems[*].cooling_system": [
            "design_sensible_cool_capacity"
        ],
        "building_segments[*].heating_ventilating_air_conditioning_systems[*].heating_system": [
            "design_capacity"
        ],
        "building_segments[*].heating_ventilating_air_conditioning_systems[*].preheat_system": [
            "design_capacity"
        ],
        "building_segments[*].zones[*].spaces[*]": [
            "floor_area",
        ],
        "building_segments[*].zones[*].surfaces[*].subsurfaces[*]": [
            "u_factor",
        ],
        "building_segments[*].zones[*].terminals[*]": [
            "served_by_heating_ventilating_air_conditioning_system"
        ],
    }
```

**Comment by claperle** [comment]
@JacksonJ-KC I'm not very familiar with this function but do you think it would be appropriate to use similar logic to what we used for assessing modeled baseline efficiency where we used whichever was available? In other words, if design capacity is available but not rated we use design capacity and vice versa. We would just have to make a decision about which to use if they are both defined.

**Comment by JacksonJ-KC** [comment]
I am thinking that design capacity should be prioritized but I don't have strong supporting arguments or a good feeling for potential pros/cons either way

**Comment by weilixu** [comment]
> I am thinking that design capacity should be prioritized but I don't have strong supporting arguments or a good feeling for potential pros/cons either way

I tend to agree with design capacity instead of rating capacity. My understanding is in ASHRAE, all mandatory efficiency requirements shall use rated capacity but design requirements (such as the one in this case) should be using design capacity.
